redash/query_runner/exasol.py

```diff
@@ -81,6 +82,7 @@ def _get_connection(self):
             dsn=exahost,
             user=self.configuration.get("user", None),
             password=self.configuration.get("password", None),
+            encryption=self.configuration.get("encrypted", True),
```

Comment:
While having SSL enabled is a good default, but this will change it for existing deployments as well. How common would it be for an Exasol deployment not to have SSL support?

Reply:
SSL support is build in into the Exasol Engine, so all deployments which support WebSockets also have SSL capabilities